.circleci/config.yml

```diff
@@ -0,0 +1,244 @@
+# Default configuration file for Drupal modules.
```

Comment:
We might want to replace this readme info with something more useful such as an overview of the setup and links to libraries/tools being used..